lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/fs.rb

```diff
@@ -137,18 +138,31 @@ def name
   #
   # Search for files.
   #
+
+  def vali_date(str)
+    date_format = '%Y-%m-%dT%H:%M:%S'
+    DateTime.strptime(str, date_format)
+    true
+  rescue
+    false
+  end
+
   def cmd_search(*args)
 
     root    = nil
     recurse = true
     globs   = []
     files   = []
+    startDate = nil
+    endDate = nil
 
     opts = Rex::Parser::Arguments.new(
       "-h" => [ false, "Help Banner" ],
       "-d" => [ true,  "The directory/drive to begin searching from. Leave empty to search all drives. (Default: #{root})" ],
       "-f" => [ true,  "A file pattern glob to search for. (e.g. *secret*.doc?)" ],
-      "-r" => [ true,  "Recursively search sub directories. (Default: #{recurse})" ]
+      "-r" => [ true,  "Recursively search sub directories. (Default: #{recurse})" ],
+      "-a" => [ true,  "Find files modified after timestamp.  Format: YYYY-mm-ddTHH:MM:SS"],
```

Comment:
I wonder if there's an easier way for users to specify dates here? i.e. Just with `YYYY-mm-dd` rather than `YYYY-mm-ddTHH:MM:SS`, or maybe even relative dates like the `find` command https://coderwall.com/p/lspgjq/list-files-modified-in-the-last-hour-find-command

Reply:
I have updated to allow YYYY-mm-dd in addition to the original YYYY-mm-ddTHH:MM:SS.